tests/end_to_end/conftest.py

```diff
@@ -7,16 +7,22 @@
 from tempfile import gettempdir
 
 from wazuh_testing.tools.file import remove_file
+from wazuh_testing import end_to_end as e2e
 
 
 alerts_json = os.path.join(gettempdir(), 'alerts.json')
 
 
 @pytest.fixture(scope='function')
-def clean_environment():
-    """Delete alerts and credentials files from the temporary folder."""
+def clean_environment(get_dashboard_credentials):
+    """Delete alerts and delete indices using the API.
```

Comment:
Only one index is deleted, right? i.e. the alerts are not deleted and then the indexes, but the index is deleted and consequently the alerts are deleted.

Reply:
@jmv74211 actually, a group of indices is deleted, as the `delete_index_api` function has `index='wazuh-alerts-4.x-*'` as a default parameter this will delete all indices that match that pattern, i.e.: wazuh-alerts-4.x-2022.06.27 and wazuh-alerts-4.x-2022.06.28

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/c5b751f480d18ab81ef619c7036bc9ab378b3436